test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -274,39 +274,68 @@ public void Should_Write_History_For_Audited_Entities_Update()
         }
 
         [Fact]
-        public void Should_Write_History_For_Audited_Entities_Update_Owned()
+        public void Should_Write_History_For_Ownded_Entities_With_Audited_Entity_As_Owner_Create()
         {
-            /* Blog has Audited attribute. */
-
-            int blog1Id;
-            var newValue = "blogger-2";
-            string originalValue;
+            // Blog has Audited attribute and it is the owner of BlogPromotion.
+            // Advertisement does not Audited attribute.
```

Comment:
This premise does not seem to affect the conclusion on the next line.

Reply:
This is to state that `BlogPromotion` has 2 foreign keys (Blog and Advertisement). However, since `Blog` is the owner, the entity history should follow `Blog` and not `Advertisement`

Reply:
Then it should be:

```c#
// Blog is the owner of BlogPromotion and has Audited attribute.
// Advertisement is not the owner of BlogPromotion.
```